binaries/Readme.md

```diff
@@ -53,15 +51,28 @@ Binaries are the artifacts generated by compiling Prisma's core (written in Rust
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-## Motivation
+## Context
```

Comment:
I personally feel that `Motivation` is a more approachable word than `Context`

Reply:
"Motivation" explains why someone is doing something. In this case either why we have "Binaries" or why we have a spec about them. The text that follows on that headline does not do that, but gives context on binaries, right? (Alternative section headline could be "Introduction")

Reply:
I prefer "introduction" over context but not strongly opinionated.